tools/bump_ocp_releases.py

```diff
@@ -172,6 +189,7 @@ def get_latest_rhcos_release_from_minor(minor_release: str, all_releases: list,
     if not all_relevant_releases:
         return None
 
+    all_relevant_releases = list(get_canonized_rhcos_versions(all_relevant_releases))
```

Comment:
maybe worth trying something like [natsort](https://pypi.org/project/natsort/) instead?
from a quick test, seems it supports such scenarios:
`print(natsorted(["4.11.0-rc.1", "4.11-fc.0", "4.11.0-0.nightly"]))`
`output: ['4.11-fc.0', '4.11.0-0.nightly', '4.11.0-rc.1']`

Reply:
Done!